packages/expo-image-picker/ios/ImagePickerModule.swift

```diff
@@ -0,0 +1,169 @@
+// Copyright 2022-present 650 Industries. All rights reserved.
+
+import UIKit
+import ExpoModulesCore
+
+typealias MediaInfo = [UIImagePickerController.InfoKey : Any]
+
+/**
+ Helper struct storing single picking operation context variables that have their own non-sharable state.
+ */
+struct PickingContext {
+  let promise: Promise
+  let options: PickingOptions
+  let imagePickerDelegate: ImagePickerDelegate
```

Comment:
One of the SwiftLint rules we have enabled is to make sure all delegates are weak (it's a common mistake among iOS devs to hold strong references to them)